src/Illuminate/Database/Eloquent/Relations/BelongsToMany.php

```diff
@@ -193,13 +193,14 @@ public function get($columns = ['*'])
      * @param  int  $perPage
      * @param  array  $columns
      * @param  string  $pageName
+     * @param  int|null  $page
```

Comment:
Both `paginate` functions should have the same docblock for `$page` argument, either `int`, either `int|null` no? Unless I miss something